website/docs/configuration/settings-reference.md

```diff
@@ -43,42 +43,43 @@ import TabItem from '@theme/TabItem';
+| `preferences.update.reminder`                 | string  | `"startup"`        | Update reminders: `"startup"` or `"never"`                                                                                                                                                                                                                                                                           |
+| `preferences.{extensionId}.engine.autostart`  | boolean | `true`             | Autostart engine on launch (e.g., `preferences.podman.engine.autostart`)                                                                                                                                                                                                                                             |
+| `proxy.enabled`                               | number  | `0`                | Proxy mode: 0=System, 1=Manual, 2=Disabled                                                                                                                                                                                                                                                                           |
+| `proxy.http`                                  | string  | `""`               | HTTP proxy URL                                                                                                                                                                                                                                                                                                       |
+| `proxy.https`                                 | string  | `""`               | HTTPS proxy URL                                                                                                                                                                                                                                                                                                      |
+| `proxy.no`                                    | string  | `""`               | No-proxy pattern (comma-separated)                                                                                                                                                                                                                                                                                   |
+| `registries.defaults`                         | array   | `[]`               | Default registries and mirrors for container image pulls. See [Managed configuration use cases](/docs/configuration/managed-configuration-use-cases#configuring-default-registries-and-mirrors) and [containers-registries.conf](https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md) |
```

Comment:
This is the change by the way!

Unfortunately formatting makes it look like the entire table was changed, it when it was just spacing being added.